src/Features/Core/Diagnostics/Analyzers/RemoveUnnecessaryImportsDiagnosticAnalyzerBase.cs

```diff
@@ -27,7 +26,11 @@ internal abstract class RemoveUnnecessaryImportsDiagnosticAnalyzerBase : Diagnos
                                                                                             isEnabledByDefault: true,
                                                                                             customTags: DiagnosticCustomTags.Unnecessary);
 
-        private static readonly DiagnosticDescriptor s_fixableIdDescriptor = new TriggerDiagnosticDescriptor(DiagnosticFixableId);
+        private static readonly DiagnosticDescriptor s_fixableIdDescriptor = new DiagnosticDescriptor(DiagnosticFixableId,
```

Comment:
Maybe add a comment about the reasons for message=null and the notconfigurable tag?